src/utils.ts

```diff
@@ -1,30 +1,25 @@
+import { PullRequest } from "./github";
+
 /**
- * @param template The backport description template
+ * @param template The template
  * @param main The main pull request that is backported
  * @param target The target branchname
- * @returns Description that can be used as the backport pull request body
+ * @returns Description that can be used in the backport pull request
  */
-export function composeBody(
+export function composeMessage(
   template: string,
-  main: PullRequest,
+  main: Pick<PullRequest, "body" | "user" | "number" | "title">,
```

Comment:
I have refactored the type here to use the existing type instead of creating a new one.

Reply:
Technically there will be no difference unless you change the type in one place and forget to do it in another one, so it is more about maintainability. From architectural point of view the same data (type) is used across the application, so there should be just one type for it. I have used `Pick` because this method does not use other attributes from the `PullRequest` type and to keep the tests for it unchanged.